<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>System_Info_Form</class>
 <widget class="QWidget" name="System_Info_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>316</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>316</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>System info</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>332</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>System name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="system_name">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>System version:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="system_version">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>System based on:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="based_on">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>System website url:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="system_website">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Kernel info:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="kernel_info">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
